reoptjl/models.py

```diff
@@ -1740,22 +1776,22 @@ class ElectricUtilityInputs(BaseModel, models.Model):
         blank=True,
         default = "Mid-case",
         help_text=("Cambium Scenario for evolution of electricity sector (see Cambium documentation for descriptions)."
-                   "Options: ['Mid-case',  'Mid-case with tax credit expiration',  'Low renewable energy cost', 'Low renewable energy cost with tax credit expiration',   'High renewable energy cost', 'High electrification',  'Low natrual gas prices', 'High natrual gas prices', 'Mid-case with 95% decarbonization by 2050',  'Mid-case with 100% decarbonization by 2035']")
+                   "Options: ['Mid-case', 'Low renewable energy cost',   'High renewable energy cost', 'High demand growth',  'Low natural gas prices', 'High natural gas prices', 'Mid-case with 95% decarbonization by 2050',  'Mid-case with 100% decarbonization by 2035']")
     )
     cambium_location_type = models.TextField(
         blank=True,
-        default = "GEA Regions",
-        help_text=("Geographic boundary at which emissions are calculated. Options: ['Nations', 'GEA Regions', 'States'].")
+        default = "GEA Regions 2023",
+        help_text=("Geographic boundary at which emissions and clean energy fraction are calculated. Options: ['Nations', 'GEA Regions 2023'].")
     )
-    cambium_metric_col = models.TextField(
+    cambium_co2_metric = models.TextField(
         blank=True,
         default = "lrmer_co2e",
         help_text=("Emissions metric used. Default is Long-run marginal emissions rate for CO2-equivalant, combined combustion and pre-combustion emissions rates. Options: See metric definitions and names in the Cambium documentation.")
     )
     cambium_start_year = models.IntegerField(
-        default=2024,
+        default=2025,
         validators=[
-            MinValueValidator(2023),
+            MinValueValidator(2025),
```

Comment:
Should we really not allow users to give a past year for this? What if we want to model a system that already exists to validate or something?

Reply:
I might be missing something, but users **can** give a year for this, it just has to be between 2025-2050 because those are the years for which Cambium has data. If they want to run a past year, they would need to supply their own custom emissions rates.

Reply:
I guess I was thinking that if cambium has time steps every 5 years from 2025 to 2050 and the nominal year also covers the 4 years preceding, that users would be allowed to provide a start year from 2021 to 2050. Maybe that's wrong though.